virttest/bootstrap.py

```diff
@@ -360,6 +365,20 @@ def _forced_or_detected(forced, detected):
         cfg.write("                            - @%s:\n" % release)
         cfg.write("                                variants:\n")
         cfg.write("                                    - @%s:\n" % arch)
+        cfg.write("variants:\n")
+        cfg.write("    - @Hostvendor:\n")
+        cfg.write("        variants:\n")
+        cfg.write("            - @%s:\n" % vendor)
+        if family:
+            cfg.write("variants:\n")
+            cfg.write("    - @Hostfamily:\n")
+            cfg.write("        variants:\n")
+            cfg.write("            - @%s:\n" % family)
+            if revision:
+                cfg.write("                variants:\n")
+                cfg.write("                    - @Hostrevision:\n")
```

Comment:
Looks `version` is more accurate than `revision`.

Reply:
sure, will change